guidance/_utils.py

```diff
@@ -272,36 +272,33 @@ def softmax(array: np.ndarray, axis: int = -1) -> np.ndarray:
     return exp_x_shifted / np.sum(exp_x_shifted, axis=axis, keepdims=True)
 
 
-class ModelStateHTMLParser(HTMLParser):
-    """Parse jupyter-flavored HTML that contains colored text to color text for the command-line"""
+# Is it good to allow user to create their own instances of output?
```

Comment:
I had a thought about allowing users to create their own parsers for the model to call

I.e. some SendToDatabase(ReadableOutput). What do you think?